### PR TITLE
feat: todos

### PR DESCRIPTION
**Pull request by priyanshu20**

### CODE REVIEW

**Comment by priyanshu20** [comment]
@ritiksr25

File: controllers/todo_controller.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		status: "pending"
+	});
+	todo = await todo.save();
+	sendSuccess(res, todo);
+};
+
+module.exports.updateTodo = async (req, res) => {
+	let { tid } = req.params;
+	let { dueDate } = req.body;
+	let todo = await Todo.findById(tid);
```
Comment (ritiksr25) [inline controllers/todo_controller.js:34]: directly use findandupdate:
`let todo = awat Todo.findOneAndUpdate({ _id: tid, uid: req.user.id }, re.body);` 

format duedate above query

File: controllers/todo_controller.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	if (dueDate) {
+		req.body.dueDate = formatHtmlDate(dueDate);
+	}
+	await todo.update({ $set: req.body }, { new: true });
+	sendSuccess(res, null);
+};
+
+module.exports.deleteTodo = async (req, res) => {
+	let { tid } = req.params;
+	let todo = await Todo.findById(tid);
```
Comment (ritiksr25) [inline controllers/todo_controller.js:54]: directly use findandDelete:
let todo = awat Todo.findOneAndDelete({ _id: tid, uid: req.user.id });

File: index.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ const runExpressServer = require("./app");
 // Check if current process is master.
 if (cluster.isMaster) {
 	// Get total CPU cores.
-	const cpuCount = os.cpus().length;
+	const cpuCount = 1;
```
Comment (ritiksr25) [inline index.js:8]: kyu bhaii :smile:

**Review by ritiksr25 — APPROVED** [review]
update the changes:)

File: index.js (hunk 3 above)
Comment (priyanshu20) [reply index.js:8]: are yeh commit hogya xD

**Review by ritiksr25 — APPROVED** [review]
naice:)

**Pull request merged**